README.md

```diff
@@ -1 +1,24 @@
-# hw01
\ No newline at end of file
+# About Me
+Hello, my name is Tianyi Qian.  
+
+### Educational Background  
+
+---
+
+I'm a grad student in the MAPSS program at the University of Chicago and I concentrate in **Quantitative Methods and Social Anlaysis**. I studied **Sociology** and **Media Studies** as an undergrad. I'm hoping to learn the following from this class:
+
+
+* _data manipulation_
+* _data visualization_
+* _get more comfortable with programming in R_
+
+### Links
+
+---
```

Comment:
Merit: You have nicely used these lines to divide your biography into sections and to partition your subheadings from the following contents.